Assets/HoloToolkit/Input/Scripts/InputSources/InputSourceEventArgs.cs

```diff
@@ -1,94 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-using System;
-using UnityEngine;
-
-namespace HoloToolkit.Unity.InputModule
-{
-    /// <summary>
-    /// Basic event args for an input source event.
-    /// </summary>
-    public class InputSourceEventArgs : EventArgs
```

Comment:
I know this is kinda unrelated to this CR, but this class reminded me of the problem that BaseInputEventData has. Because we reuse the same EventData classes in InputManager, it creates the issue that you can't cache that reference, because it could change on you.

It might make sense for BaseInputEventData to store its InputSource and SourceId in a new struct that it creates every  time time BaseInitialize is called. That way, users could easily store the source information of the event, which would make it easier to implement longer interactions such as tap-and-drag.